src/Sandbox.php

```diff
@@ -157,7 +159,10 @@ public static function fromArray(array $in): self
 
     public function toArray(): array
     {
-        $result = [];
+        $result = [
+            'branchId' => $this->branchId,
+        ];
```

Comment:
also, why treat this param differently than the others?

Reply:
It looks simpler to me this way.. I don't have to think about if the branchId is "empty" or not here, all possible values of `$this->branchId` are valid and undestood corretly by the API. This is possible because of the `if` in setter so the `$this->branchId` is never an empty string.